ui/src/app/components/admin/audit-card-list-view.tsx

```diff
@@ -0,0 +1,131 @@
+  AuditEventBundle,
+  AuditEventBundleHeader,
+  AuditTargetPropertyChange
+} from '../../../generated';
+import {ActionAuditCardBase} from '../card';
+import {FlexRow} from '../flex';
+
+
+const HidableCell = (props: {content: string}) => {
+  const {content} = props;
+  return <div style={{backgroundColor: content ? 'white' : '#f0f3f5',
+    border: '1px solid',
+    boxSizing: 'border-box'}}>{content}</div>;
+};
+
+const PropertyChangeListEntry = (props: {targetProperty?: string, previousValue?: string, newValue?: string}) => {
+  const {targetProperty, previousValue, newValue} = props;
+  return <React.Fragment>
+    <HidableCell content={targetProperty}/>
+    <HidableCell content={previousValue}/>
+    <HidableCell content={newValue}/>
+  </React.Fragment>;
+};
+
+const PropertyChangeListView = (props: { propertyChanges: AuditTargetPropertyChange[] }) => {
+  const {propertyChanges} = props;
+  const propertyCellStyle = {fontWeight: 600, border: `1px solid`};
+
+  return propertyChanges.length > 0 ?
+  <div style={{
+    margin: '0.25rem 0 0rem 1rem',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)'
+  }}>
+    <div style={propertyCellStyle}>Changed Property</div>
+    <div style={propertyCellStyle}>Previous Value</div>
+    <div style={propertyCellStyle}>New Value</div>
+    {propertyChanges.map((propertyChange, index) => <PropertyChangeListEntry {...propertyChange} key={index}/>)}
+  </div> : <div style={{margin: '0.25rem 0 0rem 1rem', fontStyle: 'italic'}}>No Property Changes</div>;
+};
+
+const typeTextColor   = colors.primary;
+const actionTextColor = colors.success;
+const targetTextColor = colors.accent;
+
+const AuditEventBundleHeaderView = (props: { header: AuditEventBundleHeader }) => {
+  const {header} = props;
+  return <div>
+    <div style={{  margin: '0',
+      display: 'flex',
+      flexDirection: 'row',
+      textAlign: 'center',
+      fontWeight: 600}}>
+      <div style={{color: typeTextColor, margin: '5px'}}>{header.agent.agentType}</div>
+      <div style={{color: actionTextColor, margin: '5px'}}>{header.actionType}</div>
+      <div style={{color: targetTextColor, margin: '5px'}}>{header.target.targetType}</div>
+    </div>
+    <div style={{
+      marginLeft: '1rem',
+      display: 'grid',
+      justifyItems: 'start',
+      columnGap: '0.5rem',
+      gridTemplateColumns: 'auto 1fr',
+      gridTemplateRows: '1fr 1fr 1fr',
+      gridAutoFlow: 'column',
+      color: typeTextColor
+    }}>
+          <div style={{fontWeight:  600}}>Agent</div>
+          <div>{`${header.agent.agentType} ${header.agent.agentId}`}</div>
+          <div>{`${header.agent.agentUsername}`}</div>
+          <div style={{fontWeight:  600, color: targetTextColor}}>Target</div>
+          <div style={{color: targetTextColor}}>{`${header.target.targetType} ${header.target.targetId || ''}`}</div>
+          <div></div>
+    </div>
+  </div >;
+};
+
+const EventBundleView = (props: { eventBundle: AuditEventBundle }) => {
```

Comment:
I don't think that the `props: ` is necessary

Reply:
I meant like:

```const EventBundleView = ({eventBundle: ActionEventBundle}) => {
[...]
```

Reply:
That looks a lot slicker, but I can't seem to get it to work.

![Screen Shot 2020-07-15 at 5 33 30 PM](https://user-images.githubusercontent.com/53479492/87598912-55e0b900-c6c1-11ea-8228-dd9cba1093a5.png)

Reply:
Ah, gotcha. Yeah, this is fine for now.